CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog for AWS IoT Device SDK for Embedded C
 
+## 202012.00 (December 2020)
+
+### Major Changes
+
+- This release introduces the **AWS IoT Over-the-air Update** client library, **backoffAlgorithm** library, and **PKCS11** library.
```

Comment:
"This release introduces the **AWS IoT Over-the-air Update** library (Release Candidate), "